File: java/com/epicdragonworld/gameserver/network/packets/sendable/EnterWorldInformation.java
package com.epicdragonworld.gameserver.network.packets.sendable;

import com.epicdragonworld.gameserver.model.actor.Player;
import com.epicdragonworld.gameserver.network.SendablePacket;

/**
 * @author Pantelis Andrianakis
 */
public class EnterWorldInformation extends SendablePacket
{
	public EnterWorldInformation(Player player)
	{
		// Packet id.
		writeShort(5);
		// TODO: Send player information.
	}
}

File: EpicDragonWorldServer/Network/Packets/Sendable/EnterWorldInformation.cs
/**
 * Author: Pantelis Andrianakis
 * Date: November 7th 2018
 */
class EnterWorldInformation : SendablePacket
{
    public EnterWorldInformation(Player player)
    {
        // Packet id.
        WriteShort(5);
        WriteLong(player.GetObjectId());
        // TODO: Send more player information.
    }
}
